.gitignore

```diff
@@ -301,5 +301,16 @@ __pycache__/
 
 ## End Visual Studio .gitignore
 
-# VS Code
-.vscode
\ No newline at end of file
+## Mac Os
+
+.Ds_Store
+
+## Testing output
+TestResults/
+CoverageReport/
+coverage.cobertura.xml
+Tools/
```

Comment:
This will exclude tools folder.

Reply:
this was just from testing my POC, please look here  https://github.com/sightmachine/azure-integrations/pull/89